src/MailPreviewMiddleware.php

```diff
@@ -56,13 +56,18 @@ private function addLinkToResponse($response, $previewPath)
         An email was just sent: <a href='".url('/themsaid/mail-preview?path='.$previewPath)."'>Preview Sent Email</a>
         </div>";
 
-        $linkHTML .= "<script type=\"text/javascript\">";
+        $timeout = intval(config('mailpreview.popup_timeout', 8000));
 
-        $linkHTML .= "setTimeout(function(){ 
-        document.body.removeChild(document.getElementById('MailPreviewDriverBox')); 
-        }, 8000);";
+        if ($timeout > 0) {
+            $linkHTML .= "<script type=\"text/javascript\">";
+
+            $linkHTML .= "setTimeout(function(){
+            document.body.removeChild(document.getElementById('MailPreviewDriverBox'));
+            }, ". intval(config('mailpreview.popup_timeout', 8000)).");";
```

Comment:
Why not reuse `$timeout` instead of grabbing from config again?

Reply:
Good point, I changed it in my last commit.